CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
+  - feat(Textarea): Implement new `Textarea` component, with new mock handlers in styleguidist
```

Comment:
💯 

I might leave the mock handlers out of the `feat(...)` changelog entry (for future commits). I think it's implied that every new component gets corresponding documentation (which includes styleguidist work, right?).